src/utils/schema/helpers/schemaHelpers.ts

```diff
@@ -20,3 +21,12 @@ export const getAllFunctionTypes = [
   ...getManyFunctionTypes,
   ...getSingleFunctionTypes,
 ];
+
+export const paginationWrapper = async <T extends Model<Document>>(
+    skip: number,
+    limit: number,
+    model: T,
+    params: any
```

Comment:
params shouldnt be any, look what type mongoos uses any copy

Reply:
Irrelevant code, deleting